eng/Generate.ps1

```diff
@@ -1,30 +1,40 @@
 #Requires -Version 6.0
-param($name, [switch]$noDebug, [switch]$reset)
+param($name, [switch]$noDebug, [switch]$reset, [switch]$noBuild, [switch]$fast)
 
 $ErrorActionPreference = 'Stop'
 
-function Invoke-AutoRest($autoRestArguments)
+function Invoke-AutoRest($baseOutput, $title, $autoRestArguments)
 {
-    $command = "npx autorest-beta $autoRestArguments"
+    $command = "npx autorest-beta $debugFlags $autoRestArguments"
```

Comment:
Use $script:debugFlags, since it isn't passed in (explicit scope access to avoid confusion).

Reply:
Done.